### ISSUE TITLE
gcov is not found in macOSX, unit tests compilation fails

### ISSUE THREAD

**Issue by lalanne** [lalanne/LoadGenerator#30]

**Comment by lalanne** [comment]
```
[ 45%] Building CXX object utst/CMakeFiles/utst.dir/Users/christian/GitHub/LoadGenerator/Requests.cpp.o
[ 54%] Building CXX object utst/CMakeFiles/utst.dir/Users/christian/GitHub/LoadGenerator/policies.cpp.o
[ 63%] Linking CXX executable utst
ld: library not found for -lgcov
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [utst/utst] Error 1
make[1]: *** [utst/CMakeFiles/utst.dir/all] Error 2
make: *** [all] Error 2

```

**Comment by lalanne** [comment]
The problem happens when compiling in tst folder, it works inside utst and itst!

**Comment by lalanne** [comment]
This does not happens anymore, I think it was a problem that was fixed as a secondary effect on #31
